tests/integration/CreateSprintTemplate.test.ts

```diff
@@ -10,28 +10,30 @@ dotenv.config();
 const username = process.env.TEST_FIXTURE_USERNAME;
 const password = process.env.TEST_FIXTURE_PASSWORD;
 
+const RUN_TESTS_IN_HEADLESS_MODE = true; // Change to 'false' in order to run tests in headful mode so you can see whats going on
```

Comment:
Yeah, this is much clearer - it's easier to see what settings one can fiddle with to adjust the testing conditions and the explanatory comments are all at the start of the file rather than crowding the actual code execution.